test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -45,3 +45,7 @@ export OVERRIDE_BAZEL_VERSION="$VERSION"
 # when running under bazel docker image, the workspace is read only.
 export OVERRIDE_BAZEL_WRAPPER_DOWNLOAD_DIR=/tmp
 bazel build -- //... "${EXCLUDED_TARGETS[@]}"
+
+cd test/distrib/bazel/cpp/
+
+bazel test //:all
```

Comment:
nit: for a test that has 2 actions (`bazel build -- //... "${EXCLUDED_TARGETS[@]}"` and `bazel test //:all`, running the in a sequence is generally speaking an antipattern  (if the first action fails, the other one will never run).
but in this case, since the first step tries to build grpc, it is perhaps reasonable to assume that if grpc build fails, there is no point of trying to build greeter on top of grpc.

Reply:
That's a good point. I can see there being cases where some (say) Android target fails to build but the distribtest would not itself fail. I'll run them all in sequence regardless of the results and report errors at the end.